package.json

```diff
@@ -973,7 +973,7 @@
   },
   "dependencies": {
     "@babel/parser": "^7.22.6",
-    "@babel/traverse": "^7.22.6",
+    "@babel/traverse": "^7.23.2",
```

Comment:
Also needs to update the mongosh dependencies, right?

Reply:
Yeah, did not add it yet to reduce changes to catch issues with CI.